library/src/components/CollapseButton.tsx

```diff
@@ -33,7 +33,7 @@ export const CollapseButton: React.FunctionComponent<Props> = ({
 }) => (
   <button
     {...rest}
-    className={`focus:outline-none ${rest.className}`}
+    className={`focus:ring-2 focus:ring-blue-500 focus:ring-offset-2 ${rest.className}`}
```

Comment:
Here instead of using `blue-500` color variant how's about using something which matches with the existing theme ?